config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -694,3 +694,52 @@ periodics:
           cpu: 2
   rerun_auth_config:
     allow_anyone: true
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-arm64
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: kind-master-arm64
+    description: Uses kubetest to run e2e tests against a latest kubernetes master created with sigs.k8s.io/kind using arm64
```

Comment:
we don't use kubetest, that description is outdated (did we miss this somewhere else?)

Reply:
fixing